### PR TITLE
labhub: Add newcomer form to developer welcome msg

### PR DESCRIPTION
**Pull request by meetmangukiya**
Closes https://github.com/coala/corobo/issues/246

# Reviewers Checklist

- [ ] Appropriate logging is done.
- [ ] Appropriate error responses.
- [ ] Handle every possible exception.
- [ ] Make sure there is a docstring in the command functions. Hint: Lookout for
  `botcmd` and `re_botcmd` decorators.
- [ ] See that 100% coverage is there.
- [ ] See to it that mocking is not done where it is not necessary.

### CODE REVIEW

File: plugins/labhub.py [hunk 1]
```diff
@@ -30,7 +30,9 @@ class LabHub(BotPlugin):
                      'also be there for you at all times to help you with '
                      'actual problems. :)',
         'developers': ' Wow @{}, you are a part of developers team now! :tada: '
-                      'Welcome to ourcommunity!',
+                      'Welcome to ourcommunity! You just passed the newcomers '
+                      'phase, we\'d like to know what could\'ve been better, '
```
Comment (SanketDG) [inline plugins/labhub.py]: "You were a newcomer before, we'd like to know..."

**Comment by meetmangukiya** [comment]
@SanketDG PTAL

File: plugins/labhub.py [hunk 2]
```diff
@@ -30,7 +30,9 @@ class LabHub(BotPlugin):
                      'also be there for you at all times to help you with '
                      'actual problems. :)',
         'developers': ' Wow @{}, you are a part of developers team now! :tada: '
-                      'Welcome to ourcommunity!',
+                      'Welcome to ourcommunity! You were a newcomer before '
+                      'we\'d like to know what could\'ve been better, please '
```
Comment (SanketDG) [inline plugins/labhub.py]: .... before, and we'd ....

**Comment by meetmangukiya** [comment]
@SanketDG PTAL

**Comment by SanketDG** [comment]
ack 4acdbda

rebase needed

**Comment by meetmangukiya** [comment]
ack b1ff0a7275cfad40c6abbff6f81169115079210c

**Comment by meetmangukiya** [comment]
ack 6a672b29f8ed759e3db7a45f2c0f45530fe2fa83

**Comment by meetmangukiya** [comment]
reack ack b1ff0a7

**Comment by meetmangukiya** [comment]
@rultor merge

**Pull request merged**